### PR TITLE
Update fixme checking

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

FIXME statements is not properly checked in pylint when they are written in
docstrings.

## Proposal

- Add a step in the circle ci config for gint-lit job to detect presence of FIXME
statements.
- Remove remaining FIXME statements
- ➕ (bonus) Update name of `run` alias in Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -245,6 +238,13 @@ run-clickhouse: ## start clickhouse backend
 	@$(COMPOSE_RUN) dockerize -wait tcp://clickhouse:9000 -timeout 60s
 .PHONY: run-clickhouse
 
+run-database: ## alias for running database services
+run-database: \
+	run-es \
+	run-mongo \
+	run-clickhouse
+.PHONY: run-database
```
Comment (jmaupetit) [inline Makefile:246]: Maybe we should include this rule as a dependency of the `run-lrs` rule?

File: Makefile [hunk 2]
```diff
@@ -224,13 +224,6 @@ logs: ## display app logs (follow mode)
 	@$(COMPOSE) logs -f app
 .PHONY: logs
 
-run: ## alias for run-es run-mongo run-clickhouse
-run: \
-	run-es \
-	run-mongo \
-	run-clickhouse
-.PHONY: run
```
Comment (jmaupetit) [inline Makefile:232]: Shouldn't we make this `run` rule an alias to `run-lrs`? To be consistent across our projects (`make run` stands for a server).

**Review by jmaupetit — COMMENTED** [review]
Great improvement, thanks! :sunglasses:

**Review by wilbrdt — COMMENTED** [review]
Alt text 
Looks good! 
The README.md would need an update on the `make run-lrs` command though

File: .circleci/config.yml [hunk 3]
```diff
@@ -488,6 +492,8 @@ workflows:
       # Check validity of git history
       - lint-git:
           filters:
+            branches:
+              ignore: add-fixme-checking
```
Comment (jmaupetit) [inline .circleci/config.yml:496]: You better configure required checks for the master branch in github.

File: Makefile [hunk 4]
```diff
@@ -245,20 +242,19 @@ run-clickhouse: ## start clickhouse backend
 	@$(COMPOSE_RUN) dockerize -wait tcp://clickhouse:9000 -timeout 60s
 .PHONY: run-clickhouse
 
+run-database: ## alias for running database services
+run-database: \
+	run-es \
+	run-mongo \
+	run-clickhouse
+.PHONY: run-database
```
Comment (jmaupetit) [inline Makefile:250]: Should be plural, _e,g,_ `run-databases`?

**Review by wilbrdt — APPROVED** [review]
Alt text

**Pull request merged**